custom-recipes/timeseries-forecast-1-train-evaluate/recipe.json

```diff
@@ -183,6 +183,7 @@
             "name": "prediction_length",
             "label": "Forecasting horizon",
             "type": "INT",
+            "description": "Number of time steps to predict future values",
```

Comment:
Number of time steps future values to predict / Number of time units future values to predict ?